helm/hpcc/templates/dafilesrv.yaml

```diff
@@ -53,6 +53,7 @@ spec:
         helmVersion: 9.2.33-closedown0
       annotations:
         checksum/config: {{ $configSHA }}
+        {{- include "hpcc.generateAnnotations" $commonCtx | indent 8 }}
```

Comment:
shouldn't these all be nindent?

Reply:
the {{- will chomp (ahead of the include function generating) so they'll be no additional blank line if there's nothing generated. But the helper function will generate a newline ahead of each new entry because the range doesn't chomp ahead of each iteration ({{- range $key, $value := $annotations }})

Example tested output:
{noformat}
      annotations:
        checksum/config: ab97c3620b03bcfc61e519ab700cb20fb8646bcf714641147b5073e682f0f910
        one: 1
        two: 2
    spec:
{noformat}